vendor/engines/badges_engine/lib/badges_engine/version.rb

```diff
@@ -0,0 +1,3 @@
+module BadgesEngine
+  VERSION = "0.0.1"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.